src/runtime/lpython_builtin.py

```diff
@@ -668,12 +668,20 @@ def pow(x: i64, y: i64, z: i64) -> i64:
 def _lpython_str_capitalize(x: str) -> str:
     if len(x) == 0:
         return x
+    i:str
+    res:str = ""
+    for i in x:
+        if ord(i) >= 65 and ord(i) <= 90:  # Check if uppercase
+            res += chr(ord(i) + 32)  # Convert to lowercase using ASCII values
+        else:
+            res += i
+
     val: i32
-    val = ord(x[0])
-    if val >= ord('a') and val <= ord('x'):
+    val = ord(res[0])
+    if val >= ord('a') and val <= ord('z'):
```

Comment:
**Line 681:** Since we already know the `ord()` value for `a` and `z`, how about we directly use it as a constant value here? For example: `if val >= 97 and val <= 122: # Check if lowercase`.

Reply:
I think using `ord('a')` is more readable.